test/testref/hofx_nomodel.ref

```diff
@@ -0,0 +1,5 @@
+Test     : Initial state: min = 271.36, max = 304.956, mean = 286.307
+Test     : Final state: min = 271.36, max = 304.956, mean = 286.307
+Test     : H(x): 
+Test     : SST nobs= 174 Min=-3.33477e+38, Max=302.17, RMS=1.33477e+38
```

Comment:
you'll notice that the observation operator is producing bad values (`-3.33477e+38`) because there are apparently observations near/on land. This will get fixed once masking is added to the interpolation.